src/main/java/org/spongepowered/common/registry/SpongeGameRegistry.java

```diff
@@ -508,13 +513,11 @@ public VillagerRegistry getVillagerRegistry() {
         return SpongeVillagerRegistry.getInstance();
     }
 
-    @SuppressWarnings("deprecation")
     @Override
     public TextSerializerFactory getTextSerializerFactory() {
         return SpongeTextSerializerFactory.INSTANCE;
     }
 
-    @SuppressWarnings("deprecation")
```

Comment:
Why remove the warnings from the drepecation bits?

Reply:
They are not used